.github/workflows/test-recent-optimism-block.yml

```diff
@@ -0,0 +1,52 @@
+name: Run a recent full Optimism block in the Hardhat Network
+
+on:
+  schedule:
+    - cron: "0 */8 * * *"
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: packages/hardhat-core
```

Comment:
I didn't notice this in the previous PR, but is this correct? Seems like a copy-paste leftover.

Reply:
Good catch. Addressed in follow-up commits